crates/core/Cargo.toml

```diff
@@ -62,7 +62,7 @@ ulid = { features = ["serde"], version = "1.1" }
 unsigned-varint = "0.7"
 wasmer = { features = ["sys"], workspace = true }
 xz2 = { version = "0.1" }
-# enum-iterator = "1.4.1"
+ecies = "0.2.6"
```

Comment:
I would actually prefer if we use RustCrypto org sanctioned libraries so they play nice with all the ecosystem and is what we have been using so far:

https://github.com/RustCrypto

includes elliptic curve libraries

Reply:
Yeah, the problem here was that I was trying to encrypt/decrypt with Elliptic Curve asymmetric crypto. While this is common with RSA asymmetric keys it's apparently not common with EC which is why I had to use a weird library.

I bit the bullet and switched to using RSA using the RustCrypto RSA implementation.